tests/unit/core/test_utils.js

```diff
@@ -957,3 +972,107 @@ describe('getClientCompatibility', () => {
     });
   });
 });
+
+describe('getCategoryColor', () => {
+  it('handles a false-y category', () => {
+    const category = null;
+    const categoryColor = getCategoryColor(category);
+
+    expect(categoryColor).toEqual(1);
+  });
+
+  it('all valid addonTypes are in CATEGORY_COLORS', () => {
+    expect(() => {
+      validAddonTypes.forEach((addonType) => {
```

Comment:
with Flow you don't need tests like this. Just sayin' :) If this loop were more expensive I'd be more worried but it's no big deal.